### PR TITLE
feat(templates): add the manufacturer to the FDO app template

### PR DESCRIPTION
**Pull request by andres-portainer**
Please review.

### CODE REVIEW

File: stacks/fdo/docker-stack.yml [hunk 1]
```diff
@@ -22,8 +22,7 @@ services:
       log4j_configuration_file: log4j2.xml
     volumes:
       - rv-target:/home/fdo/target
-    networks:
-      - fdonetwork
+    network_mode: host
```
Comment (hookenz) [inline stacks/fdo/docker-stack.yml:25]: Host networking?

File: stacks/fdo/docker-stack.yml [hunk 2]
```diff
@@ -65,13 +64,42 @@ services:
...
+  pri-fdo-manufacturer:
+    image: andresportainer/pri-fdo-manufacturer:latest
+    expose:
+      - "8039"
+    ports:
+      - "8039:8039"
+    environment:
+      manufacturer_keystore_password: RabSIZz43lympm8z
+      manufacturer_api_password: 05EV9CbHbAQANc1t
+      manufacturer_ssl_keystore-password: Sw2cqdcc1IM
```
Comment (hookenz) [inline stacks/fdo/docker-stack.yml:78]: Should these be hard coded or variables?

Comment (andres-portainer) [reply stacks/fdo/docker-stack.yml:78]: We agreed it was fine that way for the PoC, at least for now.

File: stacks/fdo/docker-stack.yml (hunk 1 above)
Comment (andres-portainer) [reply stacks/fdo/docker-stack.yml:25]: Yes, it was set up this way upstream and it makes it hard for the Owner to find the RV otherwise.

**Review by deviantony — APPROVED** [review]
LGTM

**Pull request merged**